server/game/cards/events/06/savingthekingdom.js

```diff
@@ -0,0 +1,40 @@
+const DrawCard = require('../../../drawcard.js');
+
+class SavingTheKingdom extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Make character unable to stand',
+            phase: 'challenge',
+            target: {
+                activePromptTitle: 'Select a character',
+                cardCondition: card => card.location === 'play area' && card.getType() === 'character' && card.getStrength() <= 3
+            },
+            handler: context => {
+                this.untilEndOfRound(ability => ({
```

Comment:
About the gameAction: that would prevent the whole effect right? While (simularly to Waking the Dragon), if card has multiple effects, you try to resolve as many of them so the stand prevention should still happen...